### DISCUSSION TITLE
Can obs dependency be removed for OSX?

### DISCUSSION THREAD

**Discussion by kirk86** [letmaik/pyvirtualcam#77] [Q&A] [answered]
I was wondering whether OBS can be removed as requirement from OSX?
Are you aware of `AkVCamManager` which allows to create multiple virtual cameras according to specification and stream to them via ffmpeg?

This would make the code cleaner in OSX and remove a quite large dependency IMHO.

**Comment by letmaik** [comment, marked as answer]
See https://github.com/letmaik/pyvirtualcam/issues/71

**Reply by kirk86** [reply]
Thanks for the answer, I understand this might be out of the scope of the library, if you don't mind me asking do you have any prior experience with `AkVcamManager`? If so maybe instead of adding `AkVCamManager` to the library you could just provide a python example file sending frames to `AkVCamManager`?
